samples update (#1710)
temporalio/documentation: docs-src/typescript/how-to-asynchronously-complete-an-activity-in-typescript.md

---
id: how-to-asynchronously-complete-an-activity-in-typescript

> title: How to asynchronous complete an Activity in TypeScript
> sidebar_label: Asynchronous complete an Activity
> description: Asynchronous complete an Activity in TypeScript with the complete method.

Reviewer: ```suggestion
title: How to asynchronously complete an Activity in TypeScript
sidebar_label: Asynchronously complete an Activity
description: To asynchronously complete an Activity in TypeScript, set `AsyncCompletionClient` to `complete`.
```

tags:
  - typescript
  - how-to
---

> Set the [`AsyncCompletionClient`](https://typescript.temporal.io/api/classes/client.AsyncCompletionClient) class to the [`complete`](https://typescript.temporal.io/api/classes/client.AsyncCompletionClient#complete) method, to asynchronous complete an Activity with a Task Token.

Reviewer: ```suggestion
To asynchronously complete an Activity with a Task Token, set the [`AsyncCompletionClient`](https://typescript.temporal.io/api/classes/client.AsyncCompletionClient) class to the [`complete`](https://typescript.temporal.io/api/classes/client.AsyncCompletionClient#complete) method.
```

> To asynchronously complete an Activity with a Task Token, set the [`AsyncCompletionClient`](https://typescript.temporal.io/api/classes/client.AsyncCompletionClient) class to the [`complete`](https://typescript.temporal.io/api/classes/client.AsyncCompletionClient#complete) method.

Reviewer 2: ```suggestion
To asynchronously complete an Activity, call [`AsyncCompletionClient.complete`](https://typescript.temporal.io/api/classes/client.AsyncCompletionClient#complete).
```
